kernel/src/engine/default/parquet.rs

```diff
@@ -36,16 +38,16 @@ pub struct DefaultParquetHandler<E: TaskExecutor> {
     readahead: usize,
 }
 
-/// Metadata of a data file (typically a parquet file), currently just includes the file metadata
-/// but will expand to include file statistics and other metadata in the future.
+/// Metadata of a data file (typically a parquet file).
 #[derive(Debug)]
 pub struct DataFileMetadata {
     file_meta: FileMeta,
+    stats: String,
```

Comment:
I would rather have `stats: Statistics` here and serialize to json in `as_record_batch` so that at least the parquet writer doesn't have to handle stats represented as string